test/lib/incentives-calculation.test.ts

```diff
@@ -289,15 +258,12 @@ test('correctly evaluates scenario "Single w/ $120k Household income in NJ"', as
   t.equal(taxCredits['heat_pump_water_heater'].start_date, '2023');
   t.equal(taxCredits['rooftop_solar_installation'].eligible, true);
   t.equal(taxCredits['rooftop_solar_installation'].amount.number, 0.3);
-  t.equal(taxCredits['rooftop_solar_installation'].amount.representative, 4356);
```

Comment:
This was based on some really old solar cost estimates and the number was never shown in the current calculator anyway.